src/main/java/com/group6/harmoniq/controllers/SpotifyController.java

```diff
@@ -129,12 +133,35 @@ public String callback(@RequestParam("code") String code, @RequestParam("state")
                 try {
                     Map<String, Object> body = responseEntity.getBody();
                     accessToken = (String) body.get("access_token");
+                    refreshToken = (String) body.get("refresh_token");
     
                     User user = getUserProfile(accessToken);
                     user.setTopArtist(getTopArtist(accessToken));
                     user.setTopTrack(getTopTrack(accessToken));
+                    System.out.println("SpotifyID" + user.getSpotifyId());
+                    System.out.println("DisplayName" + user.getDisplayName());
+                    System.out.println("Email" + user.getEmail());
+                    System.out.println("Followers" + user.getFollowers());
+                    System.out.println("ImageURL" + user.getImageUrl());
+                    System.out.println("TopArtist" + user.getTopArtist());
+                    System.out.println("TopTrack" + user.getTopTrack());
+                    System.out.println("externalSpotifyUrl" + user.getExternalSpotifyUrl());
```

Comment:
I think this can be removed since it doesn't serve any purpose in the app other than  debugging locally